elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideStandaloneSettings.java

```diff
@@ -235,6 +235,15 @@ default Integer getAsyncQueryCleanupDays() {
         return 7;
     }
 
+    /**
+     * Number of seconds  between async query cancellation checks.
+     *
+     * @return Default: 300
+     */
+    default Integer getAsyncQueryCancelSeconds() {
```

Comment:
I wonder if this would read better with a more descriptive name (although longer):
getAsyncQueryCancelCheckIntervalSeconds

Reply:
Maybe a better approach is to move all of the async query settings into a struct of some kind to namespace them (so they aren't so long)

Reply:
we can take care of this in the download pr

Reply:
move it into a struct